### DISCUSSION TITLE
GitHub Campus Experts Application for August 2023

### DISCUSSION THREAD

**Discussion by toukirkhan** [community/community#62368] [GitHub Education]
### Hi everyone,

I just wanted to ask when will the applications for the GitHub Campus Experts Program open for August, 2023?

**Comment by Ntshangase** [comment]
Yes they are already open, you can apply here : [campus_experts](https://education.github.com/campus_experts)

**Comment by TammyAlok2** [comment]
Hello sir At the portal of campus experts it is showing that it will closed on 1 sep 2023 23:00 but it is closed before the time . kindly help me I am unable to fill the form

**Reply by FacerAin** [reply]
I have a same problem :)  This seems to need confirmation.

**Reply by TammyAlok2** [reply]
Yes @github needs to be open the program show that we will complete our application

**Comment by dainik10** [comment]
Today is 20sep still i have not received any update of my campus expert application status mail or anything

**Reply by TammyAlok2** [reply]
I also don't get any reply
